_examples/secret/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/fastly/compute-sdk-go/fsthttp"
+	"github.com/fastly/compute-sdk-go/secretstore"
+)
+
+func main() {
+	fsthttp.ServeFunc(func(ctx context.Context, w fsthttp.ResponseWriter, r *fsthttp.Request) {
+		st, err := secretstore.Open("example_secretstore")
+		if err != nil {
+			fsthttp.Error(w, err.Error(), fsthttp.StatusBadGateway)
+			return
+		}
```

Comment:
What do you think about somehow showcasing `secretstore.ErrSecretStoreNotFound` and `secretstore.ErrSecretNotFound`? Could be something like this:

```suggestion
		switch {
		case errors.Is(err, secretstore.ErrSecretStoreNotFound):
			fsthttp.Error(w, err.Error(), fsthttp.StatusNotFound)
			return
		case err != nil:
			fsthttp.Error(w, err.Error(), fsthttp.StatusBadGateway)
			return
		}
```

Reply:
Yeah, great idea.

Reply:
Done in baed1f8